addon/components/basic-dropdown-content.ts

```diff
@@ -69,11 +71,10 @@ export default class BasicDropdownContent extends Component<Args> {
 
   @action
   setup(dropdownElement: Element): void {
-    let triggerElement = document.querySelector(`[data-ebd-id=${this.args.dropdown.uniqueId}-trigger]`);
     this.handleRootMouseDown = (e: MouseEvent | TouchEvent): any => {
-      if (e.target === null) return;
-      let target = e.target as Element;
-      if (hasMoved(e as TouchEvent, this.touchMoveEvent) || dropdownElement.contains(target) || triggerElement && triggerElement.contains(target)) {
+      let target = (e?.composedPath()[0] || e.target) as Element;
```

Comment:
`event.target` does not cross the Shadow DOM border: if the clicked element is inside Shadow DOM, then `event.target` will point at the Shadow DOM root, and the `this.args.triggerElement.contains(target)` check will fail incorrectly.

`event.composedPath()[0]` is equivalent to `event.target` but does reference the clicked element inside Shadow DOM root.

Reply:
I think that the event is always defined, but `composedPath` isn't
```suggestion
      let target = (e.composedPath?()[0] || e.target) as Element;
```

Reply:
A-a-and this broke the build and the apps that depended on this branch. 😒